app/src/main/res/values-night/themes.xml

```diff
@@ -11,6 +11,12 @@
         <item name="colorOnSecondary">@color/black</item>
         <!-- Status bar color. -->
         <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimaryVariant</item>
-        <!-- Customize your theme here. -->
+        <!-- Font family. -->
+        <!--<item name="fontFamily">@font/rubik</item>-->
+        <item name="android:fontFamily">@font/rubik</item>
+    </style>
+
+    <style name="ShapeAppearanceOverlay.App.RoundedCorners" parent="">
+        <item name="cornerSize">5dp</item>
```

Comment:
https://material.io/design/layout/spacing-methods.html#spacing
Read this and change it to even number (4dp or 6dp) - and change other items to even numbers too